<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node name="mc_to_robot" pkg="mc_to_robot_node" type="mc_to_robot_node" output="screen">
    <param name="dst_addr"  value="127.0.0.1" />
    <param name="dst_port" value="5432" />
    <param name="src_port" value="5432" />
  </node>


</launch>
